.github/pull_request_template.md

```diff
@@ -1,32 +1,48 @@
-<!-- THE FOLLOWING IS FOR THE PR AUTHOR TO FILL OUT
-PLEASE DO NOT MODIFY THE TEMPLATE BEYOND FILLING OUT THE PROPER SECTIONS -->
-## PR Author Checklist:
-<!--  Please complete all items in list. -->
-- [ ] I have linked PR's from all sub-components involved in section below. <!-- PLEASE DO NOT LINK SUBCOMPONENT ISSUES -->
-- [ ] I am confirming reviews are completed in ALL sub-component PR's.
-- [ ] I have run the full RT suite on either Hera/Hercules AND have attached the log to this PR below this line:
-  - LOG: 
-- [ ] I have added the list of all failed regression tests to "Anticipated changes" section.
-- [ ] I have filled out all sections of the template.
-
-## Description
+<!-- INSTRUCTIONS: 
+- Please fill out all sections of this PR and complete the checklist below
+- Please be as descriptive as possible, this is really important.
+- Please "fill in" checkboxes. Use [X] for a filled in checkbox or leave it [ ] for an empty checkbox
+- Please use github markup as much as possible in linking
+i.e.:
+* Linking to UFSWM PR's and issues add "- #<pr/issue number>"
+* Linking to a subcomponent PR and issues add "- <Group>/<Fork>/pull/#" or "-<Group>/<Fork>/issues/#"
+-->
+## Commit Queue Requirements:
+- [ ] Fill out all sections of this template.
+- [ ] All sub component pull requests have been reviewed by their code managers.
+- [ ] Run the full RT suite (compared to current baselines) on either Hera/Derecho/Hercules AND have committed the log to my PR branch.
+- [ ] Add list of all failed regression tests in "Regression Tests" section.
```

Comment:
```suggestion
- [ ] A list of all failed regression tests is provided in the Regression Tests section. If tests fail, I have confirmed that *only* those tests which are expected to fail do in fact fail. 
```

Reply:
@BrianCurtis-NOAA if we have rt.sh updated with capability of running full test and commit the log file, then we don't need to have this item here, right?

Reply:
Yes, that would be a change we could bring in with the PR that would make the rt.sh changes.

Reply:
I think the step to commit the logs could be automated. But I specifically want developers to verify that only the tests they expected to fail do in fact fail. It should not be up to CMs to go through the list and verify that.

If we automate the pre-test, can we add a synopsis that clarifies 
X tests failed comparison,
Y tests ran to completion (if X \=Y, then note a large message would be added)
Z compilations were completed (if Z \= expected number of compiles, then an large message added).

Reply:
Ideally the PR authors should provide 'new_baselines' file which contains the list of tests that need new baselines, and that file can then be used (by both developer(s) and code managers) to generate new baselines and verify against them. That is more useful than a log file. See #1834.

Reply:
I think it would be a mistake not to emphasize that developers must know (and report) that only the expected tests fail. They can do with a log, or maybe in a 'new baseline' file. But I've seen too many cases where logs are posted and it seems that nobody looked at them----tests didn't compile, they didn't run, or totally unexpected ones failed.